frontend/src/styles/HetComponents/HetPagination.tsx

```diff
@@ -7,15 +8,13 @@ interface HetPaginationProps {
   className?: string
 }
 
-export default function HetPagination({
-  routeConfigs,
-  className,
-}: HetPaginationProps) {
+export default function HetPagination({ routeConfigs, className }: HetPaginationProps) {
   const history = useHistory()
   const location = useLocation()
+  const [isLargeScreen, setIsLargeScreen] = useState(false)
```

Comment:
let's use the existing tailwind hooks

`const isLargeScreen = useIsBreakpointAndUp('md')`